inst/include/RcppArmadilloAs.h

```diff
@@ -403,14 +403,61 @@ namespace traits {
                     res = symmatl(res);
                 }
             }
+            else if (type == "pMatrix") {
+                IntegerVector i(ncol);
+                IntegerVector p(ncol+1);
```

Comment:
Also not sure about converting permutation matrix into a numeric matrix.

Maybe we can discuss this in another issue or PR. How about removing this for now?